pkg/testutil/test/data.go

```diff
@@ -117,16 +116,10 @@ func (dt *data) getConfig() map[ConfigKey]ConfigValue {
 }
 
 func defaultIdentifierHashing(name string) string {
```

Comment:
Why change?

Reply:
I did hit a case where the message was over 76, which got shortened to a sha256.

Turned out distribution registry rejected it (as an image name), as there is an undocumented limitation rejecting valid image names that are `64-byte hexadecimal strings`...

Reply:
I will revert this to a readable form (the sha1 are hurting readability) - it takes a bit more work though to handle the len limitation while keeping readability and avoiding collisions.